### PR TITLE
UD-933 display icons for stacks

### PR DESCRIPTION
**Pull request by akurinnoy**

### CODE REVIEW

**Comment by benoitf** [comment]
node & asp.net icons are not there

**Comment by akurinnoy** [comment]
codenvy-font.css does not contains any icon for node and asp.net

**Comment by benoitf** [comment]
codenvy-font can be generated by images that we drop in https://github.com/codenvy/che-dashboard/tree/master/src/assets/svg

so I would say that nodejs and asp.net images are required there

**Comment by slemeur** [comment]
I'll provide them in the issue

**Comment by akurinnoy** [comment]
@benoitf @ashumilova

File: src/assets/fonts/uE001-README.MD [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Images in this folder are transformed with gulp svgfonts directive
```
Comment (benoitf) [inline src/assets/fonts/uE001-README.MD]: I think we can remove this readme

**Comment by benoitf** [comment]
pls remove src/assets/fonts/uE001-README.MD (auto generated)
then +1

**Comment by ashumilova** [comment]
+1 (with composing into one commit)

**Pull request merged**